app/controllers/works_controller.rb

```diff
@@ -6,12 +6,11 @@ class WorksController < ApplicationController
   before_action :check_deposit_job_started, only: %i[show edit]
   before_action :set_work_form_from_cocina, only: %i[show edit]
   before_action :set_content, only: %i[show edit]
-  before_action :set_status, only: %i[show edit]
+  before_action :set_status, only: %i[edit]
```

Comment:
No longer needed because the new `WorkPresenter` is driving the works show view now.